integration-tests/load/ccip/ccip_test.go

```diff
@@ -0,0 +1,170 @@
+package ccip
+
+import (
+	"fmt"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
+	"github.com/smartcontractkit/chainlink-testing-framework/lib/logging"
+	"github.com/smartcontractkit/chainlink-testing-framework/wasp"
+	"github.com/smartcontractkit/chainlink/deployment"
+	ccipdeployment "github.com/smartcontractkit/chainlink/deployment/ccip"
+	tc "github.com/smartcontractkit/chainlink/integration-tests/testconfig"
+	"github.com/stretchr/testify/require"
+	"math/big"
+	"sync"
+	"testing"
+	"time"
+)
+
+var (
+	CommonTestLabels = map[string]string{
+		"branch": "ccip_load_crib",
+		"commit": "ccip_load_crib",
+	}
+	wg sync.WaitGroup
+)
+
+// step 1: setup
+// Parse the test config, initialize CRIB with configurations defined
+// step 2: load
+// Use wasp to initiate load
+// step 3: parse logs
+// Parse all events from the simulated chains, send to Loki
+// step 4: teardown
+// Stop the chains, cleanup the environment
+func TestCCIPLoad_RPS(t *testing.T) {
```

Comment:
A lot of code in this can be extracted to common functions. You might need more than one load test in future